### PR TITLE
Propagate implicitly added attributes/arguments to instantiated rules

### LINKED ISSUE #123: build_tag_filters not working as expected
Similar to https://github.com/bazelbuild/rules_docker/issues/269, it seems like tags aren't being propogated the rules generated by k8s_object:

```
//blah.replace
//blah.describe
//blah.delete
//blah.reversed
//blah.create
//blah.apply
//blah
```

E.g. given this:

```
my_custom_rule(
    name = "some_template",
    tags = ["exclude_from_tests"],
)

k8s_object(
    name = "blah",
    cluster = "my-cluster",
    kind = "deployment",
    namespace = "blahns",
    template = ":some_template",
    tags = ["exclude_from_tests"],
)
```

There is still a dependency on "my_custom_rule" even though all rules are marked as "exclude_from_tests", as the additional rules do not have the tag applied.

### PR DESCRIPTION
**Pull request by ash2k**
Fixes #123.

### CODE REVIEW

**Comment by ixdy** [comment]
can you add visibility too, as we just discussed on slack?

File: k8s/object.bzl [hunk 1]
```diff
@@ -399,24 +399,26 @@ def k8s_object(name, **kwargs):
   kwargs["image_targets"] = _deduplicate(kwargs.get("images", {}).values())
   kwargs["image_target_strings"] = _deduplicate(kwargs.get("images", {}).values())
 
+  tags = kwargs.get("tags", None)
```
Comment (ixdy) [inline k8s/object.bzl]: From https://docs.bazel.build/versions/master/skylark/lib/globals.html#rule,
> Attributes `visibility`, `deprecation`, `tags`, `testonly`, and `features` are implicitly added and cannot be overridden.

Maybe we should have an `implicit_attrs` dict listing all of these tags, and pass that as kwargs to the instantiated rules?

**Comment by ash2k** [comment]
I've made the requested changes but I don't know how to test it and I also don't have much time for it, sorry.

**Comment by mattmoor** [comment]
cc @erain this PR would be good low-risk practice for how we test PRs in this repo (b/c secrets are hard).  This is going to be easiest if I walk you through it end-to-end over GVC.

**Comment by clanstyles** [comment]
What's blocking the merge?

**Comment by erain** [comment]
Sorry @mattmoor I missed this PR and the conversation. I will take the next chance to learn from the PR tests.

**Comment by fejta** [comment]
@ash2k any chance you can head over to https://github.com/bazelbuild/rules_k8s/pull/143 and sign the CLA so we can merge this into master?

**Pull request merged**